packages/o11ytsdb/README.md

```
@@ -0,0 +1,170 @@
+node bench/run.mjs competitive
+
+# Or use Bun for ~2× faster TS execution
+bun bench/run.mjs codec
+bun bench/run.mjs competitive
+```
+
+Results are written to `bench/results/` as JSON.
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)